contrib/ci/ubuntu.sh

```diff
@@ -2,19 +2,20 @@
 set -e
 set -x
 
-#check for and install missing dependencies
-./contrib/ci/fwupd_setup_helpers.py install-dependencies --yes -o ubuntu
-
 #check we have pip
 ./contrib/ci/fwupd_setup_helpers.py install-pip --yes -o ubuntu
 
+#check for and install missing dependencies
+./contrib/ci/fwupd_setup_helpers.py install-dependencies --yes -o ubuntu
+
```

Comment:
I'm not convinced it's right to move this.  Pip isn't needed for install-dependencies command.